modules/verifying-cluster-install-oci-agent-based.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc [Using the Agent-based Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="verifying-cluster-install-oci-agent-based_{context}"]
+= Verifying that your Agent-based cluster installation runs on OCI
+
+Verify that your cluster was installed and is running effectively on {oci-first}. 
+
+.Prerequisites
+
+* You created all the required {oci} resources and services. See the "Creating OCI infrastructure resources and services" section.
+* You created `install-config.yaml` and `agent-config.yaml` configuration files. See the "Creating configuration files for installing a cluster on OCI" section.
+* You uploaded the Agent ISO image to Oracle’s default Object Storage bucket, and you created a compute instance on {oci}. See the "running-cluster-oci-agent-based" section
```

Comment:
```suggestion
* You uploaded the Agent ISO image to Oracle’s default Object Storage bucket, and you created a compute instance on {oci}. See the "running-cluster-oci-agent-based" section.
```

Reply:
As I had commented previously, not sure if we need to spell out to the reader that the referred section is in this topic?

Reply:
:heavy_check_mark: